airflow/providers/common/sql/operators/sql.py

```diff
@@ -252,13 +253,17 @@ def _process_output(self, results: list[Any], descriptions: list[Sequence[Sequen
     def execute(self, context):
         self.log.info("Executing: %s", self.sql)
         hook = self.get_db_hook()
+        if self.split_statements is not None:
+            extra_kwargs = {"split_statements": self.split_statements}
+        else:
+            extra_kwargs = {}
```

Comment:
```suggestion
            extra_kwargs = {"split_statements": self.split_statements} if self.split_statements else {}
```

Reply:
thanks for the suggestion. current form is result of TP's suggestion. just gonna leave it as is.